### PR TITLE
Improve README.md

### PR DESCRIPTION
**Pull request by takahi-i**
Hi @funwarioisii.

I improve the English in README. 

Please reviews the changes when you have a time 🙏

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,10 @@
 # decotra
 
-Save a intermediate product and upload to S3.
+This tool saves intermediate objects such as models files in machine learning project s and upload them to S3.
 
-`Decotra`'s main feature is concreted with decorator, so it is easy to code and to remove.
```
Comment (takahi-i) [inline README.md:5]: do not use contraction such as `'s` in formal documents such as README.

File: README.md [hunk 2]
```diff
@@ -1,10 +1,10 @@
...
+The main features of `Decotra` is provided with Python decorator, and therefore we can easy to add them to the codes and remove them.
 
-In my use case, tracking many results of functions on server and upload to S3 (I use minio), analyzing the calculation process ad-hoc.
```
Comment (takahi-i) [inline README.md:7]: Without conjunction, two sentences are not connected in English. The following change merged two sentences into one.

File: README.md [hunk 3]
```diff
@@ -16,9 +16,9 @@ In my use case, tracking many results of functions on server and upload to S3 (I
 $ pip install decotra
 ```
 
-and if you use local s3 server (like a minio), please set a environmental parameters `S3_ENDPOINT_URL`.
```
Comment (takahi-i) [inline README.md:19]: `like` is not formal expression. Sentences in formal documents do not start with `And` or `But`.

File: README.md [hunk 4]
```diff
@@ -16,9 +16,9 @@ In my use case, tracking many results of functions on server and upload to S3 (I
...
+When you use local s3 server (such as a minio), please set a environmental parameters `S3_ENDPOINT_URL`.
 
-Example.
```
Comment (takahi-i) [inline README.md:21]: Example is an word (not sentence).

File: README.md [hunk 5]
```diff
@@ -74,37 +74,41 @@ if __name__ == '__main__':
     main()
 ```
 
-## for developer
+## For developer
```
Comment (takahi-i) [inline README.md:77]: Some level 2 section titles start with small letters others start with capital letters in this README.

**Comment by funwarioisii** [comment]
@takahi-i 
thanks nice review!
it is very helpful (I'm not used to write formal sentence and to use English :P)

**Comment by takahi-i** [comment]
👍

**Pull request merged**